spec/fixtures/organizations.yml

```diff
@@ -3,3 +3,6 @@ acme:
 
 foo_inc:
   name: Foo Inc.
+
+deleted:
+  name: BS Corp.
```

Comment:
If you want this to represent a deleted organization, you should add the `deleted_at` attribute to the yaml. I think you can use erb to make the value be something like `1.day.ago`.